webpack/scenes/ContentViews/Details/Filters/Rules/Package/AddPackageRuleModal.js

```diff
@@ -0,0 +1,192 @@
+import React, { useState } from 'react';
+import useDeepCompareEffect from 'use-deep-compare-effect';
+import { useDispatch, useSelector } from 'react-redux';
+import { STATUS } from 'foremanReact/constants';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Modal, ModalVariant, Form, FormGroup, TextInput, ActionGroup, Button, Select, SelectOption } from '@patternfly/react-core';
+import { addCVFilterRule, getCVFilterRules } from '../../../ContentViewDetailActions';
+import {
+  selectCreatePackageFilterRule, selectCreatePackageFilterRuleError,
+  selectCreatePackageFilterRuleStatus,
+} from '../../../ContentViewDetailSelectors';
+
+const AddPackageRuleModal = ({ filterId, setIsOpen }) => {
+  const versionComparators = ['All versions', 'Equal to', 'Greater than', 'Less than', 'Range'];
+
+  const versionComparatorStaticTranslated = {
+    'All versions': __('All Versions'),
+    'Equal to': __('Equal to'),
+    'Greater than': __('Greater than'),
+    'Less than': __('Less than'),
+    Range: __('Range'),
```

Comment:
Should 'Range' be in quotes?

Reply:
Oh..It definitely should be in quotes for it to be translated as a string..It wasn't blowing up cause Range is a JS keyword.. 😕 Updated!